README.md

```diff
@@ -16,7 +16,7 @@ also able to generate compiled byte code in a C source file, see the "mrbtest"
 program under the "test" directory for an example.
 
 This achievement was sponsored by the Regional Innovation Creation R&D Programs
-of the Ministry of Economy, Trade and Industry of Japan.
+of the Ministry of Economy, Trade, and Industry of Japan.
```

Comment:
"the Ministry of Economy, Trade and Industry of Japan" is the official translation of the name of the [ministry](https://www.meti.go.jp/english/index.html). So we have to keep it.

Reply:
I was bit confused and did it by mistake i think, Indeed you are right.